components/app/account/project/ProjectDescription.vue

```diff
@@ -0,0 +1,70 @@
+<template>
```

Comment:
suggestion(non-blocking): i think we can have the whole form in one component, or do we plan to use some of the fields again? This layout just adds unnecessary complexity. I would not do this in here but create a new ticket for that.